init.d/ocvsmd.toml

```diff
@@ -14,7 +14,7 @@ unique_id = []
 # UDP has priorioty over CAN if both types are present.
 # Supported formats:
 # - 'udp://<ip4>'
-# - 'socketcan:<can_device>'
+# - 'socketcan:<can_device>' (linux only)
```

Comment:
Technically, SocketCAN is available not only on Linux -- it's also available in at least Zephyr and NuttX